### ISSUE TITLE
Optional reworded paragraphs for LexDAO landing page.

### ISSUE THREAD

**Issue by tenfinney** [lexDAO/landing-page#7]
Here is my version of the starter paragraphs using the first iteration as a template:
===================== 
LexDAO

It's a decentralized autonomous organization (DAO) of lawyers, legal professionals, software developers and engineers who embrace Web3 tools for building a new legal settlement layer.

Joining LexDAO is free and easy. The only requirements are to either be a legal professional with a law degree or currently a member of a Bar association and have desire to learn about Web3 technology, OR to be involved in Web3 technology and have a bonafide interest in leaning about the law and legal-tech and tools.

The easiest way to get to know LexDAO members is to just show up. Drop in to the Discord or Telegram and attend either the weekly community calls, and the weekly code-hacking sessions. If you contributing value and enthusiasm to the LexDAO goals you would be offered membership to the DAO. If you have heard enough already you can apply right away by providing some links to a professional or social account that represents your interest in law and technology.

Disclaimer

LexDAO is a club, not a law firm or legal entity. Unless otherwise stated in writing, all conversations and communications with DAO members which take place in any public, semi-public or online forum (e.g. Telegram, Discord, Twitter, and CryptoVoxels) are not to be considered legal advice for any purpose. To the extent applicable, LexDAO, LLC, a Delaware registered LLC, is the only legal entity able to conduct contractual business with the DAO and members of LexDAO. No individual DAO member is an agent of other DAO members or for LexDAO, LLC.

**Comment by thekerp** [comment]
I think this might be better for an about section. It's a bit wordy for what is essentially marketing copy. Shit, my copy is too wordy.

**Comment by thekerp** [comment]
Moving this over to notion, better place to jamon copy, it's here: https://www.notion.so/Hello-We-re-LexDAO-a364a7f6cb3745c2adc2b8c6a3b99c16
